emperor/support_files/js/controller.js

```diff
@@ -141,7 +192,11 @@ EmperorController.prototype.buildUI = function() {
     enableCellNavigation: true,
     forceFitColumns: true
   };
+  /**
```

Comment:
Yes, this line is correct but the lines below are missing it, I should have made that clearer.